packages/server/tsconfig.json

```diff
@@ -6,7 +6,7 @@
         "paths": {
             "@/*": ["src/*"]
         },
-        "module": "commonjs"
+        "module": "ESNext"
```

Comment:
what's the larger impact of this? Is it less compatible with servers running older versions? Or is the impact local?

Reply:
I'll create a separate build for commonjs. I want to ensure we have a good es module implementation for tree shaking

Reply:
actually, this is unnecessary as we're using `tsup` and that generates both a .cjs and .js output so we're good.